modules/openapi-generator/src/main/java/org/openapitools/codegen/utils/StringUtils.java

```diff
@@ -57,18 +75,18 @@
      * @return The underscored version of the word
      */
     public static String underscore(final String word) {
-        return underscoreWords.get(word, wordToUnderscore -> {
+        return underscoreWordsCache.get(word, wordToUnderscore -> {
             String result;
             String firstPattern = "([A-Z]+)([A-Z][a-z])";
             String secondPattern = "([a-z\\d])([A-Z])";
```

Comment:
How about deleting firstPattern + secondPattern now that you are no longer using them?

Reply:
Thanks for catching this. Done